### PR TITLE
Created a bare-bones functional site

### PR DESCRIPTION
**Pull request by mjordonez2001**
Created a site that is functional. It is completely bare-bones, just really to get the site going. Right now, it only shows a simple `<div>` with the words "Lights Out". This is a React app with Vite as a bundler.

### CODE REVIEW

File: vite.config.js
Comment (solomonhawk) [inline vite.config.js:6]: It could be interesting to take a look at the Vite Community plugins curated at [awesome-vite](https://github.com/vitejs/awesome-vite#plugins) in case anything there catches your eye. Vite also has a testing framework called [`vitest` ](https://github.com/vitest-dev/vitest) that you might be interested in taking a look at.

**Review by solomonhawk — APPROVED** [review]
LGTM (looks good to me)! 👍 Typically we try to get at least 2 approvals on PRs before merging them on client projects. Y

ou also have a choice for how to handle merging branches back into the `main` branch - manually on your local machine or through the GitHub UI (which also gives you the option to squash, rebase, or create a merge commit). This might be something you could ask Andrew about this afternoon or look for some articles on the pros/cons of each approach.

Great job!

File: .gitignore
Comment (cwmanning) [inline .gitignore]: not super important here, but it's pretty typical to add `.DS_Store` to this file so you're not tracking changes to the folder metadata from Mac OS

**Review by cwmanning — APPROVED** [review]
looks good!

File: src/assets/react.svg
Comment (nick-telsan) [inline src/assets/react.svg]: You could probably delete this file unless you plan on using it later.

**Review by nick-telsan — APPROVED** [review]
LGTM!

**Pull request merged**
